indico/modules/groups/controllers.py

```diff
@@ -54,8 +63,11 @@ def _process(self):
         return WPGroupsAdmin.render_template('groups.html', groups=groups, providers=providers, form=form,
                                              search_results=search_results, provider_titles=provider_titles)
 
+    def _check_access(self):
+        RHLocalGroupConfig._check_access(self)
```

Comment:
⚠️ Always call the methods from both base classes, otherwise you lose that functionality - so in this case, these endpoints would suddenly be completely unrestricted!

(also in the RHs below)